documentation/RLC-quickstart.md

```diff
@@ -16,96 +14,100 @@ If you are the first time to prepare the SDK, please follow the Azure SDK guidan
    The generated libraries should be in the [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js) repo, so fork and clone it in your local then the absolute path is called **${SDK_REPO_ROOT} folder**.
 
 1. Project Folder Structure.  
-   Normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules under {SDK_REPO_ROOT}. That folder will be your **${PROJECT_ROOT} folder**.  
+   Normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules under {SDK_REPO_ROOT}. That folder will be your **${PROJECT_ROOT} folder**.
 1. Package Name Convention.  
    The package name for RLC is something like `@azure-rest/{servicename}-{modulename}`. For example, the package name for Farmbeats module is `@azure-rest/agrifood-farming`.
 
-
 # How to generate RLC
 
 We are working on to automatically generate everything right now, but currently we still need some manual work to get a releasable package. Here're the steps of how to get the package.
 
-1. **Add TypeScript emitter dependency in package.json**  
-   
-   In TypeSpec project, modify `package.json` to add dependency for TypeScript emitter, then run `npm install` again to install `@azure-tools/typespec-ts`.
-
-   ```json
-   "dependencies": {
-     ...
-     "@azure-tools/typespec-ts": "latest"
-   },
+1. \*_Configure python emitter in tspconfig.yaml_
+
+   In [Azure/azure-rest-api-specs(https://github.com/Azure/azure-rest-api-specs) repo, there should be tspconfig.yaml where main.tsp of your service is. Make sure it is configured for JavaScript SDK for example:
+
+   ```yaml
+   parameters:
+     "js-sdk-folder":
+       default: "{project-root}/azure-sdk-for-js/"
+     "service-directory-name":
```

Comment:
do we support service-directory-name in the emitter now ? @MaryGao

Reply:
I don't think this parameter needs to be supported in emitter, this is part of compiler config and I think TypeSpec would take care of the logic: https://microsoft.github.io/typespec/introduction/configuration#project-parameters.